src/app/page.tsx

```diff
@@ -2,38 +2,66 @@
 import About from "@/components/sections/about";
 import FAQ from "@/components/sections/faq";
 import Schedule from "@/components/sections/schedule";
-import Team from "@/components/sections/team";
 
-import { SparklesCore } from "@/components/ui/sparkles";
+import { SparklesCore } from "@/components/sections/sparkles";
+import { TimerContainer } from '@/components/sections/TimerContainer';
+import Team from "@/components/sections/team";
 import Alt1 from "@/assets/logos/alt 1.svg";
+import { Button, MovingBorder } from "@/components/sections/moving-border";
+import { TypewriterEffectSmooth } from "@/components/sections/typewriter-effect";
+import Footer from "@/components/footer/footer";
 
 import { LazyMotion } from "framer-motion";
 
 const loadFeatures = () => import("@/lib/features").then((res) => res.default);
 
 export default function Home() {
   return (
-    <>
-      <LazyMotion features={loadFeatures}>
-        <main className="flex min-h-screen flex-col items-center justify-between md:p-24 p-8 relative">
-          <div className="w-full h-full absolute inset-0">
-            <SparklesCore
-              id="tsparticlesfullpage"
-              background="transparent"
-              minSize={0.6}
-              maxSize={1.4}
-              particleDensity={100}
-              className="w-full h-full"
-              particleColor="#FFFFFF"
-            />
+    <>    
+    <LazyMotion features={loadFeatures}>    
+    <main className="flex min-h-screen bg-black flex-col items-center justify-center p-4 sm:p-24 relative" >
```

Comment:
Remove duplicated main declaration. And uncomment the other one. Incorrect padding